engines/docker/generated_configschema.go

```diff
@@ -0,0 +1,47 @@
+// This source code file is AUTO-GENERATED by github.com/taskcluster/jsonschema2go
+
+package docker
+
+import "github.com/taskcluster/taskcluster-worker/runtime"
+
+type (
+	// Config applicable to docker engine
+	Config struct {
+
+		// Root Volume blah blah
+		RootVolume string `json:"rootVolume"`
+	}
+)
+
+func ConfigSchema() runtime.CompositeSchema {
+	schema, err := runtime.NewCompositeSchema(
+		"config",
+		`
+		{
+		  "$schema": "http://json-schema.org/draft-04/schema#",
+		  "additionalProperties": false,
+		  "description": "Config applicable to docker engine",
+		  "properties": {
+		    "rootVolume": {
+		      "description": "Root Volume blah blah",
+		      "title": "Root Volume",
+		      "type": "string"
+		    }
+		  },
+		  "required": [
+		    "rootVolume"
+		  ],
+		  "title": "Config",
+		  "type": "object"
```

Comment:
Not sure what go does here... But a stable JSON result wouldn't very useful... Maybe avoid whitespace..

CompositeSchema does a simple string comparison if two entries specifies the same schema... Maybe one day we can improve that... but stable json string wouldn't hurt either...